README.md

```diff
@@ -1,3 +1,15 @@
+# ALNS with Mabwiser
+
+The mabwiser integration isn't fully documented yet, but you can get started by looking at the example in `examples/tsp_mab.py`. The only difference from the provided `examples/travelling_salesman_problem.ipynb` is the following lines
```

Comment:
MABWiser

(please rephrase/polish a few sentences. This Readme will sit in the front of the Library repo)